_pkgdown.yml

```diff
@@ -55,6 +54,7 @@ reference:
       - col_relabel
       - example_cdisc_data
   - title: Deprecated functions and methods
-    desc: Functions deprecated in teal.data v0.3.1
+    desc: Functions deprecated in teal.data
```

Comment:
Where else would they be deprecated?
```suggestion
    desc: Deprecated functions.
```

Reply:
```
- title: Deprecated functions and methods
    desc: Deprecated functions.
```

"This looks odd, perhaps?"
```
- title: Deprecated
    desc: Deprecated  functions and methods
```

Reply:
:+1:

Reply:
Oh, actually, is `desc` required? If not, maybe just drop it?

Reply:
its not.